gravitee-node-vertx/src/main/java/io/gravitee/node/vertx/server/VertxServerOptions.java

```diff
@@ -282,8 +281,9 @@ protected final void setupTcp(TCPSSLOptions options, KeyCertOptions keyCertOptio
 
             options.setKeyCertOptions(keyCertOptions);
             options.setTrustOptions(trustOptions);
+        } else {
```

Comment:
Is is mandatory considering ssl is disabled?

Reply:
not really, when I tested, the collection was not empty so I emptied it.

Reply:
I believe it is fully ignored so I would keep it simple and not empty it.